api/admin/package.json

```diff
@@ -13,6 +13,6 @@
     "npm": ">=5.0.0"
   },
   "dependencies": {
-    "simplified-circulation-web": "0.4.30"
+    "simplified-circulation-web": "^0.4.30"
```

Comment:
Do you know why this was 0.4.30 before when package-lock.json had it at 0.4.11? Is it just that the dependencies hadn't been updated to reflect a change in package.json?

Reply:
Yes, that's what I see in the git log. @adriana-alter, am I right that we have to update `package-lock.json` because `npm` gets package versions out of it, not from `package.json`?

Reply:
Absolutely need to update `package-lock.json` (it's just a description of your currently installed package versions; it should never be out of sync with `package.json`!), but not manually; delete your local `package-lock.json` and run `npm install` to generate a new, up-to-date one.

Reply:
That's what I basically did. However, I didn't delete `package-lock.json` before but just run `npm install`.